frontend/src/App.js

```diff
@@ -79,32 +79,48 @@ class App extends Component {
       <Router>
         <div>
           <nav
+            aria-label={LOCALIZE.ariaLabel.mainMenu}
             style={styles.navBar}
             className="fixed-top bg-white navbar navbar-expand"
-            role="banner"
+            role="dialog"
```

Comment:
We'll have to test this - not sure it's the right role, but we'll leave it for now.